azure-pipelines.yml

```diff
@@ -38,7 +38,7 @@ stages:
       - task: Gradle@2
         inputs:
           options: --build-cache --stacktrace -i
-          tasks: :buildPlugin :artifactoryPublish
+          tasks: :buildPlugin :artifactoryPublish :runPluginVerifier
```

Comment:
IMO that would have been better in the validate stage. The goal of the build stage is to produce an artifact as fast as possible, and then run the rest of the pipeline in parallel, consuming the same artifact.

Reply:
You're probably right. I will make the change